docs/installation/index.md

```diff
@@ -18,19 +18,36 @@ The Autoware Foundation provides no support on other platforms than those listed
 
 Refer to [REP-2000](https://www.ros.org/reps/rep-2000.html) for the system dependencies.
 
-## Installation steps of Autoware
+## Installation of Autoware
 
-See [here](autoware) for how to install Autoware.
+There are two ways to set up Autoware. Choose one according to your preference.
 
-## Installation steps of tools for users
+### 1. Docker installation
+
+Docker can ensure that all developers in a project have a common, consistent development environment.
+It is recommended for beginners, light users, people who do not use Ubuntu.
+
+See [here](https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/installation/autoware/docker-installation.md) for the detailed steps.  
+Refer to [autowarefoundation/autoware/docker/README.md](https://github.com/autowarefoundation/autoware/blob/main/docker/README.md) as well.
+
+### 2. Source installation
+
+Source installation is for the cases where more granular control of the installation environment is needed.
+It is recommended for skilled users or people who want to customize the environment.  
+Note that some problems may occur depending on your local environment.
+
+See [here](https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/installation/autoware/source-installation.md) for the detailed steps.  
+Refer to the [autowarefoundation/autoware/README.md](https://github.com/autowarefoundation/autoware/blob/main/README.md) as well.
```

Comment:
If you write this here, it's better to remove `installation/autoware/index.md`.
It seems duplicated now.

https://autowarefoundation.github.io/autoware-documentation/pr-104/installation/autoware/

Reply:
Ok, I removed the duplication.

Note that the botton of "Autoware" don"t work. 
Do you think it is ok? 
![0515](https://user-images.githubusercontent.com/42209144/168440049-24acaae6-d7c9-4122-b037-6cfb2f49f661.jpg)

Reply:
Yes, I think it's okay. Thank you!